prismic-model/js/project-formats.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+import label from './types/label';
+
+const ProjectFormat = label('Project format');
+export default ProjectFormat;
```

Comment:
Are we sure don't want to share formats between series/stories/etc?  Do we have examples of what these might be, I usually find that useful when thinking about whether we should proliferate or not.

Reply:
Just read the ticket - my bad.